packages/solidity/contracts/StakingRewards.sol

```diff
@@ -217,6 +217,21 @@ contract StakingRewards is ILiquidityProtectionEventsSubscriber, AccessControl,
         return pendingRewards(provider, false, MAX_UINT256, liquidityProtectionStore());
     }
 
+    /**
+     * @dev returns specific provider's pending rewards for a specific participating pool
+     *
+     * @param provider the owner of the liquidity
+     * @param poolToken the pool token representing the new rewards pool
+     *
+     * @return all pending rewards
+     */
+    function pendingRewards(address provider, IERC20 poolToken) external returns (uint256) {
```

Comment:
maybe rename to `pendingPoolRewards` to prevent custom overloaded function access?

Reply:
Sure thing